src/review/routes/Error.js

```diff
@@ -14,31 +14,18 @@ const styleSheet = createStyleSheet('Error', () => ({
   },
 }))
 
-function ErrorView({ classes, error }) {
-  return (
-    <ViewContainer>
-      <ReviewAppBar />
-      <ScrollView>
-        <LayoutBody margin bottom={false}>
-          { <div className={classes.statusCode}>Error {error.statusCode}</div> }
-          { error.message && <div>{error.message}</div> }
-          { error.stack && <code><pre>{error.stack}</pre></code> }
-        </LayoutBody>
-      </ScrollView>
-    </ViewContainer>
-  )
-}
-
-ErrorView.propTypes = {
-  classes: PropTypes.object,
-  error: PropTypes.shape({
-    message: PropTypes.string,
-    stack: PropTypes.string,
-    statusCode: PropTypes.number.isRequired,
-  }),
-}
-
 export default compose(
   withStyles(styleSheet),
   connect(state => ({ error: state.data.error })),
-)(ErrorView)
+)(({ classes, error }) => (
```

Comment:
OK, it's a debug related thing, I think it should be done automatically by a webpack loader, I will give it a look.